src/components/taskDetails/DetailsColor.tsx

```diff
@@ -0,0 +1,7 @@
+const setColor: { [key: string]: string } = {
+  HIGH: 'red',
+  MEDIUM: 'orange',
+  LOW: 'green',
```

Comment:
❓Do we not want to use more visually appealing shades here?

Reply:
What do you suggest for,
1. **Red color shades**(for HIGH priority)
![image](https://user-images.githubusercontent.com/75324135/204193339-bcb5194b-f468-4add-a751-e885ceeb915c.png)

2. **Green color shades**(for LOW priority)
![image](https://user-images.githubusercontent.com/75324135/204193526-5115870b-49df-4add-b93c-9f8d4c5d7307.png)

3. **Orange color shaded**(for MEDIUM priority)
![image](https://user-images.githubusercontent.com/75324135/204193622-a2d9d41e-b933-4572-a958-0be0a89022e1.png)